packages/fetch-http-handler/src/stream-collector.ts

```diff
@@ -44,8 +44,11 @@ function readToBase64(blob: Blob): Promise<string> {
       if (reader.readyState !== 2) {
         return reject(new Error("Reader aborted too early"));
       }
-      const result = reader.result as string;
-      const dataOffset = result.indexOf(",") + 1;
+      const result = (reader.result as string) || "";
```

Comment:
Although TypeScript shows result can be `string | ArrayBuffer | null`, we're checking only for string as null as the operation name called is `readAsDataURL` on line 57  

<details>
<summary>Screenshot</summary>

<img width="716" alt="Screen Shot 2020-05-26 at 11 02 07 AM" src="https://user-images.githubusercontent.com/16024985/82935547-10660200-9f42-11ea-82ae-921447736ca8.png">

</details>